code/controllers/configuration/entries/config.dm

```diff
@@ -348,4 +348,27 @@ CONFIG_DEF(number/error_msg_delay)	// How long to wait between messaging admins
 
 CONFIG_DEF(flag/irc_announce_new_game)
 
-CONFIG_DEF(flag/debug_admin_hrefs)
\ No newline at end of file
+CONFIG_DEF(flag/debug_admin_hrefs)
+
+
+CONFIG_DEF(number/mc_tick_rate/base_mc_tick_rate)
+	integer = FALSE
+	value = 1
+
+CONFIG_DEF(number/mc_tick_rate/high_pop_mc_tick_rate)
+	integer = FALSE
+	value = 1.1
+
+CONFIG_DEF(number/mc_tick_rate/high_pop_mc_mode_amount)
+	value = 65
+
+CONFIG_DEF(number/mc_tick_rate/disable_high_pop_mc_mode_amount)
+	value = 60
+
+CONFIG_TWEAK(number/mc_tick_rate)
+	abstract_type = /datum/config_entry/number/mc_tick_rate
+
+CONFIG_TWEAK(number/mc_tick_rate/ValidateAndSet(str_val))

```

Comment:
@Cyberboss I started pushing the boundaries of the config system.

Reply:
Debating whether or not this belongs here or in config_entry.dm....

Reply:
Code organization is a balance between making a file so big it's hard to navigate and making code so split up is hard to find all the pieces.

If anything these should be split into category files of related things, sub-typed accordingly with overrides with them.